### DISCUSSION TITLE
Is it possible in spacy to detect paragraphs using NER?

### DISCUSSION THREAD

**Discussion by hacksider** [explosion/spaCy#10323] [Help: Other Questions] [answered]
So i'm having some problems here in detecting long sentences. One example is my document is around 30k characters long and my label is around 1 paragraph long. Is it possible to detect it properly?

**Comment by hacksider** [comment]
Example is
Label: Condition
Text: Means that you sought medical treatment, consultation, advice, care, or assistance, such as diagnostic measures for the condition, or symptoms related to the condition, in within three months prior to the effective date of your insurance or the effective date of an increase in your insurance

**Comment by ljvmiranda921** [comment, marked as answer]
Hi @hacksider , you can try using the [SpanCategorizer](https://spacy.io/api/spancategorizer) and see if it fits your use-case. Just note that this feature is currently experimental, but it's designed to work on long contiguous spans. For this, you need to store your entities in `doc.spans[spans_key]` instead of `doc.ents`, so you might have to process your texts a bit (cf. [storing via a SpanGroup](https://spacy.io/api/spangroup), [SpanCat vs. NER](https://prodi.gy/docs/span-categorization#ner-vs-spancat)).
